.github/workflows/pull-request-title-validation.yml

```diff
@@ -2,7 +2,10 @@ name: Validate PR Title
 
 on:
   pull_request:
-    types: [opened, edited,reopened,synchronize]
+    types:
+      - opened
+      - edited
+      - reopened
```

Comment:
This is missing the `synchronize` verb that was previously there?

Reply:
You are right. Fixing it.